terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/el/ElTest.java

```diff
@@ -270,9 +270,8 @@ public void test05_URL_Link() {
                 .getText(), is(
                         "http://localhost:8080/terasoluna-gfw-functionaltest-web/el/output_05_04?name=tera%261"));
         // inheriting of query Test
-        driver.findElement(By.id("linkUOutput")).findElement(By.linkText(
-                "http://localhost:8080/terasoluna-gfw-functionaltest-web/el/output_05_04?name=tera%261"))
-                .click();
+        driver.navigate().to(applicationContextUrl
+                + "/el/output_05_04?name=tera%261");
     }
 
     @Test(expected = NoSuchElementException.class)
```

Comment:
https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/744/files#diff-79f1423a88c365fcc49e2cd3e4f177c0R40 and etc...
- [ ] Please re-consider based #774 #773.

Reply:
The correction not related to selenium update was taken in reference to #774 #773.
https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/806/commits/fd5884433068e7bc6a3870542ec56cf60fd56f56
https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/806/commits/7676ad1d32b061b1f69669705ab8bca083469f55